src/tests/functional/plugin/conformance/test_runner/api_conformance_runner/src/behavior/ov_plugin/properties.cpp

```diff
@@ -54,4 +54,52 @@ INSTANTIATE_TEST_SUITE_P(ov_plugin_AutoBatch, OVPropertiesTests,
                 ::testing::Values(CommonTestUtils::DEVICE_BATCH),
                 ::testing::ValuesIn(ov::test::conformance::generate_ov_configs(CommonTestUtils::DEVICE_BATCH, auto_batch_properties))),
         OVPropertiesTests::getTestCaseName);
+
+const std::vector<ov::AnyMap> empty_config = {{}};
+
+INSTANTIATE_TEST_SUITE_P(ov_plugin, OVCheckChangePropComplieModleGetPropTests,
+        ::testing::Combine(
+                        ::testing::Values(ov::test::conformance::targetDevice),
+                        ::testing::ValuesIn(OVCheckChangePropComplieModleGetPropTests::getPropertiesValues()),
+                        ::testing::ValuesIn(empty_config)),
+        OVCheckChangePropComplieModleGetPropTests::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(ov_plugin_sw, OVCheckChangePropComplieModleGetPropTests,
+        ::testing::Combine(
+                        ::testing::Values(CommonTestUtils::DEVICE_HETERO, CommonTestUtils::DEVICE_AUTO, CommonTestUtils::DEVICE_MULTI),
+                        ::testing::ValuesIn(OVCheckChangePropComplieModleGetPropTests::getPropertiesValues()),
+                        ::testing::ValuesIn(ov::test::conformance::generate_ov_configs(CommonTestUtils::DEVICE_AUTO, {}))),
+        OVCheckChangePropComplieModleGetPropTests::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(ov_plugin_AutoBatch, OVCheckChangePropComplieModleGetPropTests,
+        ::testing::Combine(
+                        ::testing::Values(CommonTestUtils::DEVICE_BATCH),
+                        ::testing::ValuesIn(OVCheckChangePropComplieModleGetPropTests::getPropertiesValues()),
+                        ::testing::ValuesIn(ov::test::conformance::generate_ov_configs(CommonTestUtils::DEVICE_BATCH, {}))),
+        OVCheckChangePropComplieModleGetPropTests::getTestCaseName);
+
+const std::vector<ov::AnyMap> device_properties = {
+        {ov::device::id("0")},
+};
+
+INSTANTIATE_TEST_SUITE_P(ov_plugin, OVCheckChangePropComplieModleGetPropTests_DEVICE_ID,
+        ::testing::Combine(
+                ::testing::Values(ov::test::conformance::targetDevice),
+                ::testing::ValuesIn(device_properties),
+                ::testing::ValuesIn(empty_config)),
+        OVCheckChangePropComplieModleGetPropTests_DEVICE_ID::getTestCaseName);
+
+INSTANTIATE_TEST_SUITE_P(ov_plugin_sw, OVCheckChangePropComplieModleGetPropTests_DEVICE_ID,
+        ::testing::Combine(
+                        ::testing::Values(CommonTestUtils::DEVICE_HETERO, CommonTestUtils::DEVICE_AUTO, CommonTestUtils::DEVICE_MULTI),
+                        ::testing::ValuesIn(device_properties),
+                        ::testing::ValuesIn(ov::test::conformance::generate_ov_configs(CommonTestUtils::DEVICE_AUTO, {}))),
```

Comment:
the same

Reply:
because it is no difference in behavior of function for HETERO\MULTI\AUTO and I can put any